Fugue.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+import sys
+import subprocess
+
+class ProfileParam(object):
+    def __init__(self):
+        self._inputFile = 'input.db'
+        self._modelFile = 'model.db'
+        self._task = 'train'
+        self._topics = 100
+        self._iters = 1000
+        self._topk = 100000
+        self._LDASampler = 'binary'
+        self._random = 'deterministic'
+        self._exp = 0
+        self._log = 0
+        self._saveModel = 1
+
+    def getParams(self):
+        return_args = []
+        for key, value in self.__dict__.iteritems():
+            paramName = key.replace('_', '-')
+            return_args.append(paramName)
+            return_args.append(str(value))
+        return return_args
+
+class ProfileAP(ProfileParam, object):
+    def __init__(self):
+        super(ProfileAP, self).__init__()
+        self._inputFile = 'examples/data/ap.json'
+        self._modelFile = 'examples/models/model.ap.json'
+        self._LDASampler = 'binary'
+        self._random = 'deterministic'
+
+
+class cmdBuilder(object):
+    def __init__(self, profile):
+        self._profile = profile
+        self._other_params = []
+
+    def execute(self, all_args):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/hongliangjie/Fugue/file/2224824707/issues/source?bid=3162994&fileBranchId=3172800#l41)